### PR TITLE
Feature/typed agendapoints

### PR DESCRIPTION
**Pull request by nvdk**
type is now visually included in the agenda, would probably benefit from some styling in GN and GN-publication.

to test: 
 * make sure you've pulled dev branch of app-gelinkt-notuleren and ran the migrations.
 * use https://github.com/lblod/frontend-gelinkt-notuleren/pull/119

### CODE REVIEW

**Review by lagartoverde — APPROVED** [review]
Looks good and works good, it's a first step on refactoring the service that I think is needed.
Good job :)

**Comment by lagartoverde** [comment]
I'm finding an error using this branch for other meeting, `no concept found for uuid bdf68a65-ce15-42c8-ae1b-19eeb39e20d0` not sure what's special about this one

File: models/meeting.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import {prefixMap} from "../support/prefixes";
+import {DateTime} from 'luxon';
+import {query, sparqlEscapeString} from "mu";
+const dateFormat = process.env.DATE_FORMAT || 'dd/MM/yyyy HH:mm';
+
+export default class Meeting {
+  static async findUuid(uuid) {
```
[the remark this answers is not in the document]
Comment (nvdk) [reply models/meeting.js:7]: good point, I'll rename it to find as that's what I think I used in another model

**Comment by nvdk** [comment]
> I'm finding an error using this branch for other meeting, `no concept found for uuid bdf68a65-ce15-42c8-ae1b-19eeb39e20d0` not sure what's special about this one

hmm and did you run the migrations in app-gelinkt-notuleren? perhaps I made a typo in the frontend pr and am passing an incorrect uuid for a certain agendatype.

**Comment by lagartoverde** [comment]
It's for that, I didn't manage to run the migrations yet, not sure why it worked before maybe it was cached, if it works then I would say you can merge the code looks very clean

**Pull request merged**